.github/workflows/compilemmd.yml

```
@@ -0,0 +1,42 @@
+name: Node.js CI
+
+on:
+  push:
+    paths:
+      - "**/*.mermaid"
+      - "**/*.mmd"
+      - "**/*.md"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Use Node.js 16
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16.x
```

[must] バージョン上げちゃいますか 👀 
```suggestion
      - name: setup node 20.x
        uses: actions/setup-node@v3
        with:
          node-version: 20.x
```